webextension/background/loadSelector.js

```diff
@@ -2,6 +2,8 @@
 
 window.loadSelector = (function () {
   const scripts = [
+    "background/loadSelector.js",
```

Comment:
I thought it odd at first that we'd load loadSelector on the page to get the module names but I can't think of a better way to do it